site/_contributors/gene.txt

```diff
@@ -0,0 +1,6 @@
+---
+weight: 10
```

Comment:
Similarly here, probably makes sense to make sure this number is 1+ the last contributor